docs/interoperability.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Using Enact with third-party libaries
+---
+
+## Overview
+
+`Enact` is a fantastic way to build apps. By default we give you a template using our `Moonstone`
+library. `Moonstone` is a library designed for TVs. However, if you're not looking to create TV
+apps, you can easily use `Enact` to build applications for desktop and mobile. You can mix and match
+third-party libraries to use with `Enact` easily. Alternatively, it is also straightforward if you
+just want to use `Enact` components in your own `React` app.
+
+## Using Third-Party Components inside of Enact
+When creating an app using `enact create`, it creates an `Enact` boilerplate that setups for a
+`Moonstone` styled app. However if you want to use something outside of `Moonstone` like
+`material-ui`, `reactstrap`, or `react-router` just use `npm install`.
+
+You can include components just like you normally would by using import.
+
+```JavaScript
```

Comment:
Show command line for using a different `enact create` template?  I guess we don't have a 'core' or 'ui' template.  Wonder if we should whip one up as part of public release?

Reply:
Yeah this is a good idea. I think we should have something that is just a `ui` template.